adyenv6core/src/com/adyen/v6/facades/AdyenCheckoutFacade.java

```diff
@@ -106,9 +106,10 @@
      * In case of authorized, it places an order from cart
      *
      * @param details consisting of parameters present in response query string
-     * @return PaymentsResponse
+     * @return OrderData
+     * @throws Exception In case order failed to be created
      */
-    PaymentsResponse handleRedirectPayload(HashMap<String,String> details) throws Exception;
+    OrderData handleRedirectPayload(HashMap<String,String> details) throws Exception;
```

Comment:
Some merchants override this facade interface and create their customised implementation. I am afraid that changing the signature will result in backward incompatibility

Reply:
Will revert

Reply:
Done.